client/blocks/reader-post-card/blocked.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import SiteBlockActions from 'lib/reader-site-blocks/actions';
+import Card from 'components/card';
+import { recordTrack as recordReaderTrack } from 'reader/stats';
+import {
+	bumpStat,
+	recordGoogleEvent,
+} from 'state/analytics/actions';
+
+class PostBlocked extends React.Component {
+
+	static propTypes = {
+		post: React.PropTypes.object,
+	};
+
+	unblock = () => {
+		this.props.bumpStat( 'reader_actions', 'unblocked_blog' );
+		this.props.recordGoogleEvent( 'reader_actions', 'Clicked Unblock Site' );
+		recordReaderTrack( 'calypso_reader_unblock_site', {
+			blog_id: this.props.post.site_ID,
+		} );
+		SiteBlockActions.unblock( this.props.post.site_ID );
+	}
+
+	render() {
+		const { post, translate } = this.props;
+
+		return (
+			<Card className="reader-post-card is-blocked">
+				<p className="reader-post-card__blocked-description">
+					{ translate( 'You have blocked %(site_name)s.', { args: { site_name: post.site_name } } ) }
```

Comment:
Alternate string suggestion: 
* `translate( 'You have blocked posts from %(site_name)s.' )`&mdash; translations: **21**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/reader/site-blocked-card)